landlab/components/priority_flood_flow_router/priority_flood_flow_router.py

```diff
@@ -591,11 +590,12 @@ def _test_water_inputs(self, grid, runoff_rate):
                 grid.at_node["water__unit_flux_in"] = runoff_rate
         else:
             if runoff_rate is not None:
-                print(
-                    "FlowAccumulator found both the field "
-                    + "'water__unit_flux_in' and a provided float or "
-                    + "array for the runoff_rate argument. THE FIELD IS "
-                    + "BEING OVERWRITTEN WITH THE SUPPLIED RUNOFF_RATE!"
+                warnings.warn(
```

Comment:
@BCampforts What do you think about changing this to raise an exception rather than just a warning?

Reply:
Yes, sounds good!

Reply:
Sounds good!